core/templates/dev/head/pages/profile/Profile.js

```diff
@@ -71,13 +71,62 @@ oppia.controller('Profile', [
         }
       });
 
+      if (data.username) {
```

Comment:
(Optional) If you like, could just do:

$scope.userNotLoggedIn = !data.username;